.github/workflows/build-push-container.yml

```diff
@@ -3,8 +3,8 @@
 name: Publish Docker image on publish of release
 
 on:
-  release:
-    types: [published]
+ release:
```

Comment:
This looks inconsistent, there was 2 spaces between each tab, now it seems theres one, then two. Might be worth fixing

Reply:
nice catch, surprised the linters didn't catch it, fixed